fn_ipvoid/fn_ipvoid/lib/resilient_common.py

```diff
@@ -0,0 +1,26 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
```

Comment:
Can we rename this file to `ipvoid_helper.py` and move it to the `util` directory 

We have a library called `resilient_common` so it may be confusing